### PR TITLE
feat(pci.storages.objects): tracking implementation

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/high-pserf-storage`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7374
| License          | BSD 3-Clause

## Description
Tracking implementation for high performance storage

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.html [hunk 1]
```diff
@@ -7,7 +7,11 @@
         <oui-guide-menu
             data-text="{{:: 'pci_project_guides_header' | translate }}"
         >
-            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+            <oui-guide-menu-item
+                href="{{ $ctrl.guideUrl }}"
+                data-external
+                data-no-click="$ctrl.onGuideLinkClick()"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.html:13]:
```suggestion
                data-on-click="$ctrl.onGuideLinkClick()"
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
Changes looks good to me!

There is a typo in the commit message (`trackign` => `tracking`).
Side note: Commit message summary must use present tense.
see: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-header

File: packages/manager/apps/public-cloud/src/app.module.js [hunk 2]
```diff
@@ -122,6 +122,7 @@ export default (containerEl, environment) => {
       /* @ngInject */ ($rootScope, $state) => {
         $state.defaultErrorHandler((error) => {
           if (error.type === RejectType.ERROR) {
+            console.log(error);
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/app.module.js:125]:
```suggestion
```

File: packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js [hunk 3]
```diff
@@ -126,6 +134,13 @@ export default class PciStoragesContainersAddController {
       });
   }
 
+  trackCancel() {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js:137]: No going back ?

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**